packages/llbuild/include-llbuild-BuildSystem-BuildSystemFrontend.h.patch

```diff
@@ -0,0 +1,19 @@
+commit 111d9828e6422447321196b1826c0a666a0632d2
+Author: Igor Makarov <igormaka@gmail.com>
+Date:   Wed Aug 21 15:06:30 2019 +0300
+
+    change ProcessHandle int width so it complies with platform int width
+
+diff --git a/include/llbuild/BuildSystem/BuildSystemFrontend.h b/include/llbuild/BuildSystem/BuildSystemFrontend.h
+index b14e9c8..53bf570 100644
+--- a/include/llbuild/BuildSystem/BuildSystemFrontend.h
++++ b/include/llbuild/BuildSystem/BuildSystemFrontend.h
+@@ -116,7 +116,7 @@ public:
+   /// Handle used to communicate information about a launched process.
+   struct ProcessHandle {
+     /// Opaque ID.
+-    uintptr_t id;
++    uint64_t id;
```

Comment:
The first iteration of this pull wasn't building for 32-bit targets on the CI, so I backported this fix.